application/helpers/admin/import_helper.php

```diff
@@ -2357,13 +2357,16 @@ function XMLImportSurvey($sFullFilePath, $sXMLdata = null, $sNewSurveyName = nul
             if (!isset($insertdata['quotals_quota_id']) || (int)$insertdata['quotals_quota_id'] < 1) {
                 continue;
             }
-            $insertdata['autoload_url'] = 0; // used to bypass urlValidator check in QuotaLanguageSetting model
             $insertdata['quotals_quota_id'] = $aQuotaReplacements[(int) $insertdata['quotals_quota_id']]; // remap the qid
             unset($insertdata['quotals_id']);
-            $quotaLanguagesSetting = new QuotaLanguageSetting();
+            $quotaLanguagesSetting = new QuotaLanguageSetting('import');
             $quotaLanguagesSetting->setAttributes($insertdata, false);
             if (!$quotaLanguagesSetting->save()) {
-                throw new Exception(gT("Error") . ": Failed to insert data<br />");
+                $header = sprintf(gT("Unable to insert quota language settings for quota %s"), $insertdata['quotals_quota_id']);
```

Comment:
We will only get here when the validator is used, as it has been before. That means with the "import" scenario that has been implemented, the user will never get a hint, that the urls are not set properly.

Reply:
PS2 : see the bad current workaround 
https://github.com/LimeSurvey/LimeSurvey/blob/9ad92b3d08f168e486a59c5ad589c9bb110cc0ac/application/helpers/admin/import_helper.php#L2360

The current comment is clear enough